backend/src/api/mentors/mentors.api.ts

```diff
@@ -24,11 +24,12 @@ const initMentorsApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
     ) {
       const { courseId, userId } = req.body;
 
-      const courseToMentor = await coursesToMentorsService.createMentorToCourse(
-        { courseId, userId },
-      );
+      const addMentorTocourse = await mentorService.addMentorToCourse({
```

Comment:
I think the result of adding mentor to course should be named something like that
```suggestion
      const addedToCourseMentor = await mentorService.addMentorToCourse({
```